doc/endpoint/integrations.md

```diff
@@ -66,6 +66,18 @@ The iron codecs contain a validator which apply the constraint to validated valu
 
 Similarly to `tapir-refined`, you can find the predicate logic in `integrations/iron/src/main/scala/sttp/iron/codec/iron/TapirCodecIron.scala` and provide your own given `ValidatorForPredicate[T, P]` in scope using `ValidatorForPredicate.fromPrimitiveValidator`
 
+When using `tapir-iron` in the server, `tapir-iron-server` might become useful.
+It defines custom exception and `ironFailureHandler` and `ironDecodeFailureInterceptor` to use
+in server options.
```

Comment:
a code snippet on how to use this would be nice, can be with the netty interpreter :)